openmc/data/neutron.py

```diff
@@ -817,6 +833,29 @@ def from_njoy(cls, filename, temperatures=None, evaluation=None, **kwargs):
             ev = evaluation if evaluation is not None else Evaluation(filename)
             if (1, 458) in ev.section:
                 data.fission_energy = FissionEnergyRelease.from_endf(ev, data)
+                # Add 318 fission heating data from heatr
+                non_fission_heating = Reaction(999)
+                non_fission_heating.redundant = True
+                fission_heating = Reaction(318)
+
+                heatr_evals = get_evaluations(kwargs["heatr"])
+                for heatr in heatr_evals:
+                    temp = "{}K".format(round(heatr.target["temperature"]))
+                    f318 = StringIO(heatr.section[3, 318])
+                    get_head_record(f318)
+                    _params, fission_kerma = get_tab1_record(f318)
+                    fission_heating.xs[temp] = fission_kerma
+                    total_heating_xs = data.reactions[301].xs.get(temp)
+                    if total_heating_xs is None:
+                        continue
+                    non_fission_heating.xs[temp] = Tabulated1D(
+                        fission_kerma.x,
+                        total_heating_xs(fission_kerma.x) - fission_kerma.y,
+                        breakpoints=fission_kerma.breakpoints,
+                        interpolation=fission_kerma.interpolation)
```

Comment:
NJOY linearizes cross sections so that they are meant to be lin-lin interpolable. I think it's safe to just omit the explicit settings of `breakpoints` and `interpolation`.